source/developers/quickstarts/setup_a_backend_dev_environment.rst

```diff
@@ -1,2 +1,298 @@
+  pip install -e "./tutor[full]"
+
+If you are using ARM-64, then install this extra plugin and enable it
+(if you're not, then skip this step):
+
+.. code-block:: bash
+
+  pip install git+https://github.com/open-craft/tutor-contrib-arm64
+  tutor plugins enable arm64
+  tutor config save
+
+Finally, let's configure and provision your Open edX instance!
+You will be asked a couple questions.
+Answer them however you like, although the default answers will work fine.
+
+.. code-block:: bash
+
+  tutor dev quickstart
+
+Depending on your system and your Internet connection speed,
+this could take anywhere from five minutes to over an hour,
+so go get a coffee and come back for the next part.
+
+
+Working with a fork
+*******************
+
+At this point you should have a Tutor installation that is suitable for
+development, but you're still missing a practical way to edit the code, test
+it locally, and then contribute it back.
+
+For the purposes of this tutorial, you'll be modifying code in the
+``_edx-platform`` repository, where the Open edX backend code lives.  Let's
+start by creating your own personal "fork" of it. A "fork" is essentially your
+own copy of the repository. `See here <https://docs.github.com/en/get-started/quickstart/fork-a-repo>`_ to learn more about forks.
+
+**Forking edx-platform**
+
+Assuming you're logged in to Github, forking a repository is easy.  Visit the
+``edx-platform`` repository at this URL:
+
+https://github.com/openedx/edx-platform
+
+Now, click the :guilabel:`Fork` button on the top right, and in the next
+screen, select your personal account as the owner.  After you click the
+:guilabel:`Create fork` button, you'll be taken to your own version of the
+``edx-platform`` repository.
+
+**Cloning your fork**
+
+Your ``edx-platform`` currently only exists in the Github servers.  You'll now
+create a local copy of it (a "clone").
+
+First, fetch the git URL of your fork.  Navigate to its web page (to which you
+were taken after creation), click on the :guilabel:`Code` button, select
+the **HTTPS** tab, and copy the URL given.  It should look like this:
+
+``https://github.com:<your_github_username>/edx-platform.git``
+
+Now, from the same top level directory you created above, clone the repository
+as follows:
+
+.. code-block:: bash
+
+   cd ~/openedx
+   git clone  https://github.com:<your_github_username>/edx-platform.git
+
+You'll now have an ``edx-platform`` directory containing a local clone of your
+fork.  It is not yet wired into your Tutor development environment, though.
+This is what you'll do next.
+
+**Mounting edx-platform**
+
+To have Tutor run your local fork of edx-platform, you have to tell it to do so
+on start up.  It is a simple CLI parameter that points Tutor to the directory where
+the code lives.
+
+As a first step, fire up a one-off LMS container while mounting your local
+checkout:
+
+.. code-block:: bash
+
+   tutor dev run --mount=${PWD}/edx-platform lms bash
```

Comment:
Is it clear what PWD means? (just thinking from the perspective of someone hitting a lot of this for the first time, things can be kind of confusing when you're jamming in a bunch of new knowledge) cc @arbrandes

Reply:
I guess this is part of the discussions about assumptions.  `$PWD` is reasonably basic command-line stuff.

Reply:
Ya, that's why I asked. This one feels squishy but I guess I have no strong thoughts.

Reply:
I think it's fine to leave, they're trusting us to run a lot of commands already.